--- pom.xml	2024-10-22 22:33:49.384835600 +0200
+++ "pom.xml"	2024-10-22 22:34:18.154175600 +0200
@@ -124,6 +124,11 @@
 					<descriptorRefs>
 						<descriptorRef>jar-with-dependencies</descriptorRef>
 					</descriptorRefs>
+					<archive>
+					    <manifest>
+					        <mainClass>net.runelite.cache.CollisionMapDumper</mainClass>
+					    </manifest>
+					</archive>
 				</configuration>
 				<executions>
 					<execution>
